extension/chrome/settings/index.htm

```diff
@@ -66,7 +66,6 @@
             <b class="red">Following features are disabled for <span class="email-address"></span>:</b>
```

Comment:
Eventually we should remove the whole `auth_denied_warning`. I'll make another issue for that.